roles/foreman_installer/tasks/module_prs.yml

```diff
@@ -10,7 +10,7 @@
     executable: /opt/puppetlabs/puppet/bin/gem
   with_items:
     - puppet-strings
-  when: puppetlabs_gem.stat.exists
+  when: ( puppetlabs_gem.stat.exists and determine_puppetlabs_gem_check is not defined )
```

Comment:
I'd prefer to exclude the whole file if it isn't needed. I don't see where `determine_puppetlabs_gem_check` would come from.

Reply:
for now it is just two variables that i set when calling the playbook
````
  import_playbook: 'forklift/playbooks/katello.yml'
  vars:
          determine_puppetlabs_gem_check: true
          no_ostree_repo: true
````

Reply:
Why do you find you encounter this? This task file should only be included if oyu are specifying PRs or special branches for testing per:

https://github.com/theforeman/forklift/blob/4612ea43361e61f94ca75b3dba8f843997944578/roles/foreman_installer/tasks/main.yml#L7-L8

Reply:
Probably because they were changed to arrays so the check no longer works as intended.

Reply:
https://github.com/theforeman/forklift/pull/885